ui/raidboss/data/05-shb/raid/e8n.js

```diff
@@ -395,34 +394,34 @@
       },
       'replaceText': {
         'The Path of Light': 'Voie de lumière',
-        'Twin Stillness': 'Entaille de la quiétude',
+        'Twin Stillness/Twin Silence': 'Entaille de la quiétude/Entaille de la tranquilité',
         'Stoneskin': 'Cuirasse',
-        'Spiteful Dance': 'Danse de la froideur',
+        'Spiteful Dance/Embittered Dance': 'Danse de la froideur/Danse de la sévérité',
```

Comment:
Yes. But I just understood the confusion. This is because of certain lines in other files like e8s.
`39.3 "Biting/Driving Frost" sync /:Shiva:4D6[67]:/` Lines like this are shortened on the name, it generates an error and it gives: `'Biting/taillade de givre',`  

I don't know for other languages, but you have to translate the line as it is written in the txt file, otherwise the display is incorrect. I've been fixing this for a while on all files.

In the above case where the names are whole, this works normally. For the sake of consistency, can we leave it as it is written and translated from the text file?

Reply:
That would be a question for quisquous, I think. My understanding is that we want to translate individual skills individually if possible. This goes both for this and for the situation on E7n.

Reply:
@quisquous, what do tou think for this situation e7n, e8, ?

Reply:
Yeah, I agree with @JLGarber here.  I think it's preferable to not add additional translation lines where they're not necessary.  If the individual skills cover the translations, then that's simplest, in my opinion.

Reply:
Thinking about it throughout the day, I agree with you 2. As explained above, when the names are whole in the line of the txt file, it works well and translates all the other lines in any order. But otherwise as on e8s or other, when the names are shortened, it does not work anymore and I would have to translate line by line. Thank you for this interesting discussion.